docs/api/client/hoodie.account.rst

```diff
@@ -20,11 +20,11 @@ Example
 
 .. code:: js
 
-    if (hoodie.account.isSignedIn()) {
-    renderWelcome(hoodie.account)
-    }
+    hoodie.account.get('username').then(function (username) {
+      renderWelcome(username);
```

Comment:
`hoodie.account.get('username')` will always resolve, but if you are signed in it will resolve with a username, otherwise with undefined. Also there are valid cases when a user is authenticated but has no username property, it’s better to use `account.get('session')`

```js
account.get('session').then(function (sessionProperties) {
  if (!sessionProperties) return redirectToHome()
  renderWelcome(sessionProperties)
})
```

Does that make sense?